docs/Using-Fleet/REST-API.md

```diff
@@ -1047,6 +1038,7 @@ Modifies the Fleet's configuration with the supplied information.
 | deadline                          | string  | body  | _mdm.macos_updates settings_. Hosts that belong to no team and are enrolled into Fleet's MDM won't be able to dismiss the Nudge window once this deadline is past. **Requires Fleet Premium license** |
 | custom_settings                   | list    | body  | _mdm.macos_settings settings_. Hosts that belong to no team and are enrolled into Fleet's MDM will have those custom profiles applied. |
 | enable_disk_encryption            | boolean | body  | _mdm.macos_settings settings_. Hosts that belong to no team and are enrolled into Fleet's MDM will have disk encryption enabled if set to true. **Requires Fleet Premium license** |
+| enable_end_user_authentication            | boolean | body  | _mdm.macos_setup settings_. If set to true, end user authentication will be required during automatic MDM enrollment of new macOS devices. End user authentication must be configured separately. **Requires Fleet Premium license** |
```

Comment:
I wonder if we could be more clear about what "End user authentication" is or where it can be configured. You're way better at words than me.

I'm thinking something like

> Settings for your IdP provider must be configured separately in `mdm.end_user_authentication`